Packages/manifest.json

```diff
@@ -3,10 +3,10 @@
     "com.unity.ads": "2.3.1",
     "com.unity.analytics": "3.2.2",
     "com.unity.collab-proxy": "1.2.15",
-    "com.unity.package-manager-ui": "2.0.3",
+    "com.unity.package-manager-ui": "2.0.7",
```

Comment:
Probably wanna leave the manifest.json and ProjectVersion.txt changes out of this one (I assume these should generally be updated in their own smaller change, unless the other changes in this PR require this update)